_data/simple-icons.json

```
@@ -2455,6 +2455,11 @@
             "source": "https://brand.clubhouse.io",
             "guidelines": "https://brand.clubhouse.io"
         },
+        {
+            "title": "Clubhouse2",
```

Pending discussion on #9329 - this can be updated to just `Clubhouse`.
```suggestion
            "title": "Clubhouse",
```